### PR TITLE
Feature/ros2 update mobilitypath visualizer

### PR DESCRIPTION
**Pull request by adev4a**
# PR Details
Ports mobilitypath_visualizer to ros2
## How Has This Been Tested?
Unit tested and locally integration tested

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: mobilitypath_visualizer/src/mobilitypath_visualizer.cpp
Comment (JonSmet) [inline mobilitypath_visualizer/src/mobilitypath_visualizer.cpp:210]: It might be helpful to include a comment here that `rclcpp::Duration(0.1 * 1e9)` is 0.1 seconds.

**Review by JonSmet — CHANGES_REQUESTED** [review]

File: mobilitypath_visualizer/src/mobilitypath_visualizer.cpp
Comment (JonSmet) [inline mobilitypath_visualizer/src/mobilitypath_visualizer.cpp:63]: Since the specific parameter .yaml file is used when launching this node from environment.launch.py now (rather than this parameter being a global parameter), I believe we can remove the forward-slash here and the parameter can simply be read in as `"vehicle_id"`

File: mobilitypath_visualizer/config/params.yaml
Comment (JonSmet) [inline mobilitypath_visualizer/config/params.yaml:31]: This `host_id` parameter is actually loaded in from the `vehicle_id` parameter included in the /opt/carma/vehicle/calibration/identifiers/UniqueVehicleParams.yaml file. Since this file is passed at launch time from guidance.launch.py, we can remove the parameter from here.

File: mobilitypath_visualizer/test/test_mobility_path_visualizer.cpp
Comment (JonSmet) [inline mobilitypath_visualizer/test/test_mobility_path_visualizer.cpp:2]: Let's keep it as 2019-2022 and I think there are some other files (like package.xml) that still need to be edited as well.

File: mobilitypath_visualizer/src/mobilitypath_visualizer.cpp
Comment (JonSmet) [inline mobilitypath_visualizer/src/mobilitypath_visualizer.cpp:65]: This can be simplified to just `RCLCPP_INFO_STREAM(get_logger(), "Loaded params: " << config_);` since the overloaded output stream was implemented in mobilitypath_visualizer_config.hpp

File: mobilitypath_visualizer/src/mobilitypath_visualizer.cpp
Comment (JonSmet) [inline mobilitypath_visualizer/src/mobilitypath_visualizer.cpp:47]: We'll need to add `config_.host_id` here as well

**Review by JonSmet — CHANGES_REQUESTED** [review]

**Comment by msmcconnell** [comment]
Wait on merging this after approval. I'll merge it directly into develop when I'm ready.

**Review by JonSmet — APPROVED** [review]

**Comment by adev4a** [comment]
This PR requires a corresponding change in carma-config which needs to be merged first in order to avoid issues at launch https://github.com/usdot-fhwa-stol/carma-config/pull/162

**Comment by msmcconnell** [comment]
I'm good with this being merged now, whenever you guys are ready. But please double check the build now that develop is building stably again.

**Pull request merged**
